bedrock/newsletter/views.py

```diff
@@ -113,8 +115,18 @@ def confirm_thanks(request):
     return l10n_utils.render(request, "newsletter/confirm.html", {"token_error": token_error, "generic_error": generic_error}, ftl_files=FTL_FILES)
 
 
+def newsletter_all_json(request):
+    """Returns a JSON string of all newsletters configured in Basket."""
+    data = {"newsletters": get_newsletters()}
+    return HttpResponse(json.dumps(data), content_type="application/json")
```

Comment:
You do have `JsonResponse` already imported up there. Which does most everything for you. It could be just:

```python
return JsonResponse({"newsletters": get_newsletters()})
```